packages/noya-utils/src/__tests__/sortBy.test.ts

```diff
@@ -3,3 +3,83 @@ import { sortBy } from '../index';
 test('empty array', () => {
   expect(sortBy([], '')).toEqual([]);
 });
+
+test('first key', () => {
+  expect(
+    sortBy(
+      [
```

Comment:
If this is an array we use multiple times, we can move it outside of the `test` function to reuse it.

Reply:
where would we place this array?

Reply:
Outside of every function, so e.g.

```ts
const names = [{ name: 'Liam' }, ...]

test('first key', () => {
  expect(sortBy(names, 'name'))...
})

test('second key', () => {
  expect(sortBy(names, 'last'))...
})
```

Reply:
Got it!